apps/wallet/src/ui/app/redux/RootReducer.ts

```diff
@@ -4,13 +4,15 @@
 import { combineReducers } from '@reduxjs/toolkit';
 
 import app from './slices/app';
+import coins from './slices/coins';
 import permissions from './slices/permissions';
 import transactionRequests from './slices/transaction-requests';
 
 const rootReducer = combineReducers({
 	app,
 	permissions,
 	transactionRequests,
+	coins,
```

Comment:
We shouldn't be increasing our redux usage, we should be removing this. To fix this across usages we either need to listen to storage change events, or use something like zustand or jotai with a storage write.

Reply:
switched to use zustand